src/operations/device/query.getDeviceById.js

```diff
@@ -20,6 +23,9 @@ const getDeviceById = async (root, { deviceId }, {token}) => {
         device.attrs.push({
           label: attr.label,
           valueType: formatValueType(attr.value_type),
+          id: attr.id,
+          isDynamic: attr.type === 'dynamic',
```

Comment:
I think that just return the `attr.type` here would be better 🤔
It can be done in the other queries too, but in other PRs.